src/webapi/history.rs

```diff
@@ -0,0 +1,64 @@
+#![cfg(feature = "serde")]
+use webcore::value::Reference;
+use webcore::try_from::TryInto;
+use webcore::serialization::JsSerializable;
+
+/// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/History)
+pub struct History(Reference);
+
+reference_boilerplate! {
+    History,
+    instanceof History
+}
+
+impl History {
+    /// `History.pushState` allows you to add history entries.
```

Comment:
In general we don't reference the original JS `camelCased` name in the docs, and there is no point in referencing the name of the function you're documenting anyway. (: